server/src/models/activities/activity.controller.ts

```diff
@@ -53,7 +53,7 @@ export class ActivityController {
     const { name, price, isArchived } = activityData;
     return this.activityService.updateActivity({
       where: { id },
-      data: { name, price, isArchived, updatedAt: new Date() },
+      data: { name, price: +price, isArchived, updatedAt: new Date() },
```

Comment:
I believe we don't need to specify [`updatedAt`](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#updatedat)

Reply:
This was what was here but yes I just removed it